examples/preprocessing/plot_css.py

```diff
@@ -0,0 +1,96 @@
+"""
+=================================================================
+Cortical Signal Suppression (CSS) for removal of cortical signals
+=================================================================
+
+This script shows an easy example of how to use CSS
```

Comment:
pedagogically, it's best not to claim that examples are "easy" because it subtly shames a user who struggles to understand it or get it to work (same goes for the word "just", as in "you just need to...")